beta/nncf/tensorflow/batchnorm_adaptation.py

```diff
@@ -0,0 +1,32 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from nncf.common.batchnorm_adaptation import BatchnormAdaptationAlgorithmImpl
+from nncf.common.utils.logger import logger as nncf_logger
+
+
+class TFBatchnormAdaptationAlgorithmImpl(BatchnormAdaptationAlgorithmImpl):
+    """
+    Implementation of the batch-norm adaptation algorithm for the TensorFlow backend.
```

Comment:
```suggestion
    Implementation of the batch-norm statistics adaptation algorithm for the TensorFlow backend.
```
The 'batch-norm statistics adaptation' is used in the documentation. Please, update here and below.